tensorflow/tensorflow-api/build.gradle

```diff
@@ -5,7 +5,8 @@ configurations {
 }
 
 dependencies {
-    api "org.bytedeco:javacpp:1.5.4"
+    // javacpp bug fix https://github.com/bytedeco/javacpp/commit/7f27899578dfa18e22738a3dd49701e1806b464a
+    api "org.bytedeco:javacpp:1.5.6-SNAPSHOT"
```

Comment:
When will this package becomes non-snapshot?

Reply:
I asked in the issue https://github.com/awslabs/djl/issues/690 but Samuel didn't answer it. will follow up with him